src/renderer/catalog-entities/kubernetes-cluster.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import React from "react";
+import { categoryVersion } from "../../common/catalog";
+import type { CatalogCategorySpec, CatalogEntityConstructor } from "../../common/catalog";
+import { KubernetesCluster, KubernetesClusterCategory as KubernetesClusterCommonCategory } from "../../common/catalog-entities";
+import styles from "./kubernetes-cluster.module.scss";
+
+export class KubernetesClusterCategory extends KubernetesClusterCommonCategory {
```

Comment:
The other option could be to have `displayColumns` be an optional parameter to the category's constructor. That way we don't need a new base class.

Another option would be to expand the `categoryColumnsInjectable` with an injection token for internal registrations.